pandas/core/generic.py

```diff
@@ -6220,9 +6220,9 @@ def replace(self, to_replace=None, value=None, inplace=False, limit=None,
         8             4.71
         dtype: object
 
-        Filling in `NaN` in a Series via polynomial interpolation or splines:
-        Both `polynomial` and `spline` methods require that you also specify
-        an `order` (int).
+        Filling in ``NaN`` in a Series via polynomial interpolation or splines:
+        Both 'polynomial' and 'spline' methods require that you also specify
+        an ``order`` (int).
```

Comment:
Arguments should be single backticks - double is only for literals right and code samples I think

Reply:
My understanding is that double backticks is for code, including parts like a single variable `None`, an assignment `foo=1`... Single backticks is for things that you can refer (link) to, like a function, class, module... And for values just quotes.

For an argument, I'd consider it more code, that something you can link to. That's why I added double backticks. But it's very subtle, I'd be happy with any option (no quoting, single backticks, double backticks and quotes).

Does this make sense?

Reply:
Yep thanks. I think I've seen other instances where parameters are in single backticks but this is nuanced enough that it shouldn't hold up the PR - can be part of a larger conversation.

Reply:
I added a bullet point to #20298 to decide a standard for these cases. I think at the moment there is not much consistency.